editor/store/test/selectors.js

```diff
@@ -2616,224 +2639,335 @@ describe( 'selectors', () => {
-			expect( getInserterItems( {}, false ) ).toEqual( [] );
-		} );
-	} );
-
-	describe( 'getFrecentInserterItems', () => {
-		beforeAll( () => {
-			registerCoreBlocks();
-		} );
-
-		it( 'should return the most frecently used blocks', () => {
+		it( 'should set isDisabled when a block with useOnce has been used', () => {
 			const state = {
-				preferences: {
-					insertUsage: {
-						'core/deleted-block': { time: 1000, count: 10, insert: { name: 'core/deleted-block' } }, // Deleted blocks should be filtered out
-						'core/block/456': { time: 1000, count: 4, insert: { name: 'core/block', ref: 456 } }, // Deleted shared blocks should be filtered out
-						'core/image': { time: 1000, count: 3, insert: { name: 'core/image' } },
-						'core/block/123': { time: 1000, count: 5, insert: { name: 'core/block', ref: 123 } },
-						'core/paragraph': { time: 1000, count: 2, insert: { name: 'core/paragraph' } },
-					},
-				},
 				editor: {
 					present: {
 						blocksByUID: {
-							carrot: { name: 'core/test-block' },
+							block1: { name: 'core/test-block-b' },
+						},
+						blockOrder: {
+							'': [ 'block1' ],
 						},
-						blockOrder: [],
 						edits: {},
 					},
 				},
 				sharedBlocks: {
-					data: {
-						123: { uid: 'carrot' },
-					},
+					data: {},
 				},
 				currentPost: {},
+				preferences: {
+					insertUsage: {},
+				},
+				blockListSettings: {},
+				settings: {},
 			};
-
-			expect( getFrecentInserterItems( state, true, 3 ) ).toMatchObject( [
-				{ name: 'core/block', initialAttributes: { ref: 123 } },
-				{ name: 'core/image', initialAttributes: {} },
-				{ name: 'core/paragraph', initialAttributes: {} },
-			] );
+			const items = getInserterItems( state );
+			const testBlockBItem = items.find( ( item ) => item.id === 'core/test-block-b' );
+			expect( testBlockBItem.isDisabled ).toBe( true );
 		} );
 
-		it( 'should weight by time', () => {
+		it( 'should give common blocks a medium utility', () => {
 			const state = {
-				preferences: {
-					insertUsage: {
-						'core/image': { time: Date.now() - 1000, count: 2, insert: { name: 'core/image' } },
-						'core/paragraph': { time: Date.now() - 4000, count: 3, insert: { name: 'core/paragraph' } },
-					},
-				},
 				editor: {
 					present: {
-						blockOrder: [],
+						blocksByUID: {},
+						blockOrder: {},
+						edits: {},
 					},
 				},
 				sharedBlocks: {
 					data: {},
 				},
+				currentPost: {},
+				preferences: {
+					insertUsage: {},
+				},
+				blockListSettings: {},
+				settings: {},
 			};
-
-			expect( getFrecentInserterItems( state, true, 2 ) ).toMatchObject( [
-				{ name: 'core/image', initialAttributes: {} },
-				{ name: 'core/paragraph', initialAttributes: {} },
-			] );
+			const items = getInserterItems( state );
+			const testBlockBItem = items.find( ( item ) => item.id === 'core/test-block-b' );
+			expect( testBlockBItem.utility ).toBe( 1 );
```

Comment:
Should we use constant `MEDIUM` instead of value `1` to better match with the description of the test?

Reply:
👍 added some constants in 6efa3afd2